src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
+			if ( $has_plus ) :
+				esc_html_e( 'Thanks for installing Event Tickets Plus! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			else :
+				esc_html_e( 'Thanks for installing Event Tickets! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			endif;
+		?></p>
 	</div>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-sos"><?php esc_html_e( 'Resources and Support', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'We’ve got your back every step of the way.', 'event-tickets' ); ?></p>
-		<ul>
-			<li><a href="http://m.tri.be/1a9w" target="_blank"><?php esc_html_e( 'Search the Knowledgebase', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9x" target="_blank"><?php esc_html_e( 'Available Translations', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9y" target="_blank"><?php esc_html_e( 'Submit a Help Desk Request', 'event-tickets' ); ?></a></li>
+	<div class="tribe-events-admin-quick-nav">
+		<div class="tribe-events-admin-quick-nav__title"><?php esc_html_e( 'Quick Links:', 'event-tickets' ); ?></div>
+		<ul class="tribe-events-admin-quick-nav__links">
+			<?php if ( $has_plus && $has_tec ) : // ET+ with TEC ?>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="edit.php?page=tribe-common&tab=event-tickets&post_type=tribe_events" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Configure Settings', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=woocommerce&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install WooCommerce', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="post-new.php?post_type=tribe_events" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Create Ticket', 'event-tickets' ); ?></a></li>
+			<?php elseif ( $has_plus ) : // ET+ without TEC ?>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="edit.php?page=tribe-common&tab=event-tickets" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Configure Settings', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=woocommerce&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install WooCommerce', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=the-events-calendar&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install The Events Calendar', 'event-tickets' ); ?></a></li>
+			<?php elseif ( $has_tec ) : // ET with TEC ?>
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 600fecb54bfb0086c2a32fb5249858c4</sub>